portfolio/src/main/webapp/life-map.js

```diff
@@ -0,0 +1,93 @@
+  let birthContent = '<div id="content">'+
```

Comment:
All of these variables can be const if you're not changing them.

Reply:
Change to const